app/uk/gov/hmrc/tariffclassificationfrontend/service/ReportingService.scala

```diff
@@ -39,8 +40,19 @@ class ReportingService @Inject()(connector: BindingTariffClassificationConnector
     connector.generateReport(report)
   }
 
+  def getQueueReport(implicit hc: HeaderCarrier): Future[Seq[ReportResult]] = {
+    val statuses = Set(NEW, OPEN, REFERRED, SUSPENDED).map(_.toString)
+
+    val report = CaseReport(
+      filter = CaseReportFilter(status = Some(statuses), assigneeId = Some("none")),
```

Comment:
This looks a bit odd - can't we use assigneeId = None, rather than Some("none")

Reply:
maybe looks odd because you are mixing None (optional scala object) and none (which means, select cases with no assigned operator) :-)